### PR TITLE
Using Location instead if SiriLocation

### PR DESCRIPTION
**Pull request by YoSev**
Some bug fixes
Added support for "where is <city/country>"

The "fixed" Version of Location is required first! This won't work without this fix..

### CODE REVIEW

**Comment by ghost** [comment]
my springboard crashes :( how can this be resolved?

**Comment by YoSev** [comment]
No more.. As i mentioned in the description, the Location Class had to be updated, but this is done now..

**Comment by ghost** [comment]
Thanks. Two questions. Where can I get the fixed location file? How do I update the server if I am on a mac? Cheers :)

**Comment by YoSev** [comment]
it's in siriObject/systemObjects.py
you either use git (that calls you installed siriServer using git) or you just download the file and replace it..

**Comment by ghost** [comment]
ok thanks. I have a domain. I set up the server on the domain by using sudo git clone blah blah. It works perfectly for me but other people connecting to it say that no plugins work. Can only the person (me) who is controlling the server get the server to respond properly? When I generated the certs, I used the 192.etc... one. Should I have used the domain.com one?

**Comment by YoSev** [comment]
Yes!

**Comment by ghost** [comment]
Its ok got it sorted. They didnt change the language to US lol

**Comment by YoSev** [comment]
That's needed too, yes.. (or de-DE)

File: plugins/whereAmI.py [hunk 1]
```diff
@@ -10,12 +10,10 @@
 from plugin import *
  
 from siriObjects.baseObjects import AceObject, ClientBoundCommand
-from siriObjects.systemObjects import GetRequestOrigin
+from siriObjects.systemObjects import GetRequestOrigin, Location
 from siriObjects.uiObjects import AddViews, AssistantUtteranceView
-from siriObjects.mapObjects import SiriLocation, SiriMapItem, SiriMapItemSnippet
+from siriObjects.mapObjects import SiriMapItem, SiriMapItemSnippet
```
Comment (janrueth) [inline plugins/whereAmI.py:15]: There are also correct objects for this I will soon remove mapObjects.py, please switch to "localsearchObjects.py" objects.

But bare in mind commit [1a93c7e53c48374c47a6e026369b2033d89ca29f](https://github.com/Eichhoernchen/SiriServer/commit/1a93c7e53c48374c47a6e026369b2033d89ca29f)

**Comment by janrueth** [comment]
sorry this repo is not maintained anymore, no pulls

**Pull request closed without merging**